ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm

```diff
@@ -0,0 +1,14 @@
+Value SERVICE_ID (\d+)
+Value TYPE (\S+)
+Value ADMIN_STATUS (\S+)
+Value OPERATIONAL_STATUS (\S+)
+Value CUSTOMER_ID (\d+)
+Value SERVICE_NAME (\S+)
+
+Start
+  ^ServiceId\s+Type\s+Adm\s+Opr\s+CustomerId\s+Service\s+Name -> Services
+
+Services
+  ^\s*${SERVICE_ID}\s+${TYPE}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s+${CUSTOMER_ID}\s+${SERVICE_NAME}\s*$$ -> Record
+  ^-+\s*$$
+  ^Matching Services : \d+\s*$$ -> End
```

Comment:
If you like this suggestion, click the Commit suggestion button to accept my edit. Thank you!

```suggestion
  ^Matching Services\s+:\s+\d+\s*$$ -> End
```